### ISSUE TITLE
It's MODBUS!

### ISSUE THREAD

**Issue by warhammerkid** [warhammerkid/bluetti_mqtt#47]
I don't know why it took me so long to realize this, but the command/response API is just MODBUS over Bluetooth (and MODBUS over MQTT for the cloud support over WiFi). The internal API should be reworked to use standard terminology, as well as proper exception handling.

**Comment by Marko298** [comment]
Does it mean we can connect somewhere on the board to modbus?

**Comment by warhammerkid** [comment]
@Marko298 Sorry, I don’t really know the answer to that. Taking my AC300 apart is a bit outside my comfort zone.

**Comment by mark-b-ab** [comment]
It has communication interface exposed right below the DC input

**Comment by tinkercnc** [comment]
@mark-b-ab OK, and do you know the pinout?

**Comment by Marko298** [comment]
> @mark-b-ab OK, and do you know the pinout?

Nope, I can try to probe with logical analyzer. But I will be able to do it only in a week or so

**Comment by tinkercnc** [comment]
I'm certainly looking forward. :)

**Comment by Patrick762** [comment]
If I understood the device data correctly, there should be only 2 variants of address ranges. The only question is, which device uses which variant. With that info we could make this software much simpler to include new models of powerstations
